code/online_challenges/src/project_euler/problem_026/problem_026.cpp

```diff
@@ -0,0 +1,41 @@
+#include <iostream>
+#include <vector>
+int main()
```

Comment:
I would add a newline before main for separation. Otherwise, good job! 👍

Reply:
Thanks